### ISSUE TITLE
Error occurred when executing ImageResize+

### ISSUE THREAD

**Issue by wsry888** [cubiq/ComfyUI_essentials#47]
Error occurred when executing ImageResize+:

'bool' object has no attribute 'startswith'

File "D:\SD\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "D:\SD\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "D:\SD\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "D:\SD\ComfyUI\custom_nodes\ComfyUI_essentials\image.py", line 288, in execute
elif method.startswith('fill'):

**Comment by quill18** [comment]
I just encountered this error as well and I suspect you probably were doing the same thing as me: Importing an existing workflow that used ImageResize+. For me, the issue was that the "Method" parameter was set to "False". I think the workflow I imported may have been referencing an older version of ImageResize+ that didn't have this parameter. 

"False" isn't a valid option for "Method". Clicking on and setting this parameter to one of the actual allowed values (like "stretch") fixed the problem for me.
